src/client/layouts/default.vue

```diff
@@ -10,12 +15,16 @@
 
 <script>
   import { AppFooter, AppHeader } from '../components/'
+  import menuI18n from '../static/data/menu.json'
+  import app from '../static/data/app.json'
```

Comment:
I'd like it if we can prevent `app` from being statically imported. This means it is being added to every JS bundle of every page. Maybe we can reuse the `title` from the Menu model in the CMS. We can discuss this in person.